js/main.js

```diff
@@ -50,6 +51,46 @@ $entryForm.addEventListener('submit', function () {
   data.editing = null;
 });
 
+// here start the delete process
+
+var $deleteTag = document.querySelector('.deleteTag');
+var $modalHolder = document.querySelector('.modalHolder');
+var $cancel = document.querySelector('.cancel');
+var $confirm = document.querySelector('.confirm');
+var modal = true;
```

Comment:
I would start this variable with a value of `false`, since the modal isn't open when the page loads.